src/components/header/index.css

```diff
@@ -413,6 +403,110 @@
 .user-notes p {
   padding: 20px 0;
 }
+
+.share-content a {
+  color: #000;
+}
+
+.share-content p {
+  line-height: 0.7;
+}
+
+.exported-url {
+  display: flex;
+}
+
+.exported-url span:first-child {
+  width: 100%;
+  overflow: hidden;
+  vertical-align: middle;
+  line-height: 25px;
+  white-space: nowrap;
+}
+
+.exported-url a::selection {
+  background: rgba(0, 0, 0, 0.15);
+}
+
+.share-buttons,
```

Comment:
We are still not sharing the style between the data viewer and the modals, right?